typescript/introduction/tsc-data-types-i/tsc-array-tuple.md

```diff
@@ -0,0 +1,119 @@
+---
+author: kapnobatai136
+
+type: normal
+
+category: must-know
+
+aspects:
+  - introduction
+
+---
+
+# Array & Tuple
+
+---
+## Content
+
+### Array
+
+An array represents a collection of values. In TypeScript, there are two ways in which you can define an array:
+
+```ts
+// by putting the [] after the type
+// here we define an array of numbers
+let numberArray: number[] = [1, 3, 5, 7];
+
+// or by wraping your type in the Array type
+// here we define an array of strings
+let stringArray: Array<string> = [
+  "one",
+  "three",
+  "five",
+  "seven"
+];
+```
+
+Note that arrays can only hold values of the type that you specify.
+
+If you want to store more than one type, you can use a union:
+
+```ts
+// array of strings and numbers
+let mixedArray: Array<string|number>[] = [
```

Comment:
```suggestion
let mixedArray: Array<string|number> = [
```

Reply:
Typescript playground allows you to add code in URL https://www.typescriptlang.org/play/#code/DYUwLgBAtglgHiAJgQQE6oIYE8BcE2ZYA8AzmKjAHYDmAPpQK5QBGIqAfBALwQDaAUBAgBGADQRBEAOQB7SiCnjJAZiVCArGukAzGADcF-ALoBufkA